CodeEditModules/Modules/CodeFile/src/Model/CodeLanguage.swift

```diff
@@ -0,0 +1,217 @@
+//
+//  CodeLanguage.swift
+//  
+//
+//  Created by Henry Chu on 25/3/2022.
+//
+
+import Foundation.NSURL
+
+/// The language of a code codument.
+public struct CodeLanguage {
+    /// The language name used in highlightr.
+    let name: String
```

Comment:
I like this solution however thoughts on having a custom type for the name. It's not really good to pass around strings if we have to do any type of comparison or logic. Maybe this could be a phantom type or we can just use the Language enum

Reply:
Yeah thanks for the suggestions. I brought back the original `Language` enum as `HightlightrLanguage`. See if you are fine with the changes.